community/ndp/runtime/src/test/java/org/neo4j/ndp/runtime/integration/RecordingCallback.java

```diff
@@ -40,7 +40,7 @@
 
     public Call next() throws InterruptedException
     {
-        Call msg = calls.poll( 10, TimeUnit.SECONDS );
+        Call msg = calls.poll( 10000, TimeUnit.SECONDS );
```

Comment:
10 seconds? 10_000 seconds? The exception message indicates this is 10 seconds. But this method call are telling 10_000 seconds...